app/app-upgrade-client/src/Login.tsx

```diff
@@ -87,60 +87,62 @@ export const Login: FC = () => {
   }
 
   return (
-    <Container className={classes.root}>
```

Comment:
Kind of hard to read the diff here. Was anything changed? Or Was this just wrapped in a <React.Fragment /> for some reason.

Reply:
The fragment was added when I had the nav on each page rather than at the top level. Fragment removed.